diff --git a/node_modules/gatsby-transformer-documentationjs/gatsby-node.js b/node_modules/gatsby-transformer-documentationjs/gatsby-node.js
index 965d8cf..9b09324 100644
--- a/node_modules/gatsby-transformer-documentationjs/gatsby-node.js
+++ b/node_modules/gatsby-transformer-documentationjs/gatsby-node.js
@@ -286,7 +286,7 @@ exports.onCreateNode = async ({
       const docSkeletonNode = {
         commentNumber,
         level,
-        id: createNodeId(docId(node.id, docsJson)),
+        id: createNodeId(docId(parent, docsJson)),
         parent,
         children: [],
         internal: {
